charts/infra-apps/Chart.yaml

```diff
@@ -21,3 +21,8 @@ annotations:
       links:
```

Comment:
I guess you have to delete the old changes to only have the last entry in here with this specific change/update

Reply:
oh really?

Reply:
well as it is versionned it makes total sense, lemme update this